linode/manager
# refactor: [M3-6263] - MUI v5 Migration - SRC > Features > EntityTransfers

## Description 📝
MUI v5 CSS migration for SRC > Features > EntityTransfers

## Major Changes 🔄
- Removed `React.FC` or `
- Renamed interface `Props` to `{ComponentName}Props`.
- Removed `CombinedProps` if it's directly referencing `Props` with nothing additional
- Removed `makeStyles` for new `styled API`.
- `export` component inline and removed `export default {ComponentName}` at bottom of file. 

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-08-23 at 4 46 02 PM](https://github.com/linode/manager/assets/139489745/da6192f3-b011-428f-a25e-194d0db8fb91) | ![Screenshot 2023-08-23 at 4 46 10 PM](https://github.com/linode/manager/assets/139489745/a285e059-509e-4fa8-be1d-76f86e74c4ca) |
| ![Screenshot 2023-08-23 at 4 47 10 PM](https://github.com/linode/manager/assets/139489745/7c2029c6-a58c-4706-91b4-690bc459efa7) | ![Screenshot 2023-08-23 at 4 47 22 PM](https://github.com/linode/manager/assets/139489745/8637cd2b-4a9d-4f9c-80d4-52eac128db40) |
| ![Screenshot 2023-08-23 at 4 47 42 PM](https://github.com/linode/manager/assets/139489745/b055820b-2a4e-4e00-a2fd-c7ee52f05283) | ![Screenshot 2023-08-23 at 4 48 07 PM](https://github.com/linode/manager/assets/139489745/dadedbd7-ff0a-48bd-8223-f3bc64efaa8b) |

## How to test 🧪
1. Navigate to http://localhost:3000/account/service-transfers
2. Ensure the functionality and styling is consistent
